crates/rmf_site_editor/src/site/save.rs

```diff
@@ -1759,6 +1761,39 @@ pub fn generate_site(
     });
 }
 
+pub fn export_grid(world: &mut World, path: &PathBuf) {
+    let mut system_state: SystemState<(Query<(&Grid, &ChildOf)>, Query<&NameInSite>)> =
+        SystemState::new(world);
+    let (grids, names) = system_state.get(&world);
+    let mut i = 0;
+    for (grid, parent) in grids {
+        let mut img = image::RgbImage::new(
+            (grid.range.width() + 1) as u32,
+            (grid.range.height() + 1) as u32,
+        );
+
+        img.fill(255u8);
+        let Ok(name) = names.get(parent.0) else {
+            error!("Could not get level name");
+            continue;
+        };
+
+        for cell in &grid.occupied {
+            img.put_pixel(
+                (cell.x - grid.range.min_cell().x) as u32,
+                (cell.y - grid.range.min_cell().y) as u32,
+                image::Rgb([0, 0, 0]),
+            );
+        }
+
+        let mut path = path.clone();
+        path.push(format!("occupancy.{}.png", name.0));
+        img.save(path).unwrap();
```

Comment:
I would avoid unwrapping here, I suppose the main failure scenario here would be an invalid `PathBuf` passed to the function which should not really happen but again catching the error and logging it + continuing it would be a safer behavior